Microsoft.AVS.VVOLS/Microsoft.AVS.VVOLS.psd1

```diff
@@ -0,0 +1,136 @@
+    # Script module or binary module file associated with this manifest.
+    RootModule        = 'Microsoft.AVS.VVOLS.psm1'
+
+    # Version number of this module.
+    ModuleVersion     = '1.0.0'
+
+    # Supported PSEditions
+    # CompatiblePSEditions = @()
+
+    # ID used to uniquely identify this module
+    GUID              = '5fa1a15b-4953-4eba-9d69-1cf8d7a94f70'
+
+    # Author of this module
+    Author            = ''
+
+    # Company or vendor of this module
+    CompanyName       = ''
+
+    # Copyright statement for this module
+    Copyright         = ''
+
+    # Description of the functionality provided by this module
+    Description       = 'Azure VMware Solutions VMFS Package (Automation only).'
+
+    # Minimum version of the PowerShell engine required by this module
+    PowerShellVersion = '5.1'
+
+    # Name of the PowerShell host required by this module
+    # PowerShellHostName = ''
+
+    # Minimum version of the PowerShell host required by this module
+    # PowerShellHostVersion = ''
+
+    # Minimum version of Microsoft .NET Framework required by this module. This prerequisite is valid for the PowerShell Desktop edition only.
+    # DotNetFrameworkVersion = ''
+
+    # Minimum version of the common language runtime (CLR) required by this module. This prerequisite is valid for the PowerShell Desktop edition only.
+    # ClrVersion = ''
+
+    # Processor architecture (None, X86, Amd64) required by this module
+    # ProcessorArchitecture = ''
+
+    # Modules that must be imported into the global environment prior to importing this module
+    RequiredModules   = @(
+        @{ "ModuleName" = "Microsoft.AVS.Management"; "ModuleVersion" = "5.2.39" }
+    )
+
+    # Assemblies that must be loaded prior to importing this module
+    # RequiredAssemblies = @()
+
+    # Script files (.ps1) that are run in the caller's environment prior to importing this module.
+    # ScriptsToProcess = @()
+
+    # Type files (.ps1xml) to be loaded when importing this module
+    # TypesToProcess = @()
+
+    # Format files (.ps1xml) to be loaded when importing this module
+    # FormatsToProcess = @()
+
+    # Modules to import as nested modules of the module specified in RootModule/ModuleToProcess
+    # NestedModules = @()
+
+    # Functions to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no functions to export.
+    FunctionsToExport = @(
+        "New-VvolDatastore",
+        “Remove-VvolDatastore”,
+        "New-VvolVasaProvider",
+        "Update-VMHostCertificate",
+        "Remove-VvolVasaProvider",
+        "New-VvolStoragePolicy",
+        "Remove-VvolStoragePolicy"
+    )
+
+    # Cmdlets to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no cmdlets to export.
+    CmdletsToExport   = @()
+
+    # Variables to export from this module
+    VariablesToExport = '*'
```

Comment:
Do we need any variables exported? Didn't seem so.

Reply:
We do not need it. I removed it